flow_action_components/CollectionProcessors/force-app/main/default/classes/SortCollection.cls

```diff
@@ -2,37 +2,35 @@
  * Provides a Salesforce Ligntning Flow component for sorting collections.
```

Comment:
This is great! It's one thing when someone submits new features. That's always nice. But it's _really_ great when someone hardens an existing extension. Thanks for the contribution!